Tools/wasm/README.md

```diff
@@ -23,16 +23,24 @@ to a repository checkout.
 Christian Heimes maintains a container image with Emscripten SDK, Python
 build dependencies, WASI-SDK, wasmtime, and several additional tools.
 
+From within your local CPython repo clone, run one of the following commands:
+
 ```
 # Fedora, RHEL, CentOS
-podman run --rm -ti -v $(pwd):/python-wasm/cpython:Z quay.io/tiran/cpythonbuild:emsdk3
+podman run --rm -ti -v $(pwd):/python-wasm/cpython:Z -w /python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
 
 # other
-docker run --rm -ti -v $(pwd):/python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
+docker run --rm -ti -v $(pwd):/python-wasm/cpython -w /python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
 ```
 
+Note that when you exit the container, all work will be lost.
+See the `podman` or `docker` docs for how to preserve the container environment
+across runs.
+
```

Comment:
The work is not lost when you exit the container. With the correct setup the work is inside the CPython checkout.

```suggestion
```

Reply:
We specify docker options that explicitly remove the container after exit. Should we change this, then?

Reply:
```
$ docker run --help | grep "\-\-rm"
      --rm                             Automatically remove the container when it exits
```

Reply:
Yes, that is correct. ``--rm`` removes the container so we don't end up with tons of dead container instances. All build artifacts are persisted in the mounted directory.

Reply:
Are there further information missing regarding what the "correct setup" is? :)

Reply:
Is 7db408ceb6b2c0c9e7859ea1b63d1a3ca76693e8 acceptable?

Reply:
> Are there further information missing regarding what the "correct setup" is? :)

You tell me :) Do the instructions work for you? 

I have run the containers hundreds times. The commands are embedded in my muscle memory. I didn't even notice that the instructions were missing the CWD because it was so obvious to me.

Reply:
I think we're good now :)